### PR TITLE
OI-125,142 Changed moderation content view title, and altered local t…

### PR DESCRIPTION
**Pull request by nvelychenko**
…ask title. Changed slideshow arrows.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -239,6 +239,7 @@
         "drupal/votingapi_reaction": "^1.1",
         "drupal/votingapi_widgets": "1.x-dev",
         "npm-asset/masonry-layout": "^4.2",
+        "npm-asset/moment": "^2.29",
```
Comment (db-Matroskeen) [inline composer.json]: @nvelychenko, please take a look: https://momentjs.com/docs/#/-project-status

**Review by db-Matroskeen — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (nvelychenko) [reply composer.json]: @db-Matroskeen I removed unneeded moment library.

File: modules/openideal_challenge/openideal_challenge.module [hunk 2]
```diff
@@ -23,6 +25,19 @@ function openideal_challenge_cron() {
...
 }
 
+/**
+ * Implements hook_form_BASE_FORM_ID_alter().
+ */
+function openideal_challenge_form_node_form_alter(&$form, FormStateInterface $form_state, $form_id) {
+  if (isset($form['created']['widget'][0]['value'])) {
+    $value = &$form['created']['widget'][0]['value'];
+    $date_format = DateFormat::load('openideal_date')->getPattern();
+    $value['#attributes']['data-date-format'] = 'DD/MM/YYYY';
```
Comment (db-Matroskeen) [inline modules/openideal_challenge/openideal_challenge.module]: Is there any way to use the format from field settings?

Comment (nvelychenko) [reply modules/openideal_challenge/openideal_challenge.module]: I didn't found it. There is #date_time_format, but it works only for Safari.

**Pull request merged**